lib/github.sh

```diff
@@ -205,6 +203,21 @@ strap::github::api::token::create() {
     strap::abort "$msg"
   fi
 
+  #Test if we need to do an SSO enable
```

Comment:
Do we want to do the test after we save the token below? Just in case something in terms of user interaction causes the strap script to abort while the thing is paused, the actual token itself will be lost and become useless.